log/options.go

```diff
@@ -83,13 +81,6 @@ func TimeFormatOption(format string) Option {
 	}
 }
 
-// TraceOption add option to enable/disable print of stacktrace on error log
```

Comment:
We cannot remove this option, we are on v1. Is that really an issue? It if off by default, so it doesn't get in the way anyway.

Reply:
ill revert